DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -127,6 +127,7 @@ public DiscordMember Owner
         /// Gets the guild's voice region.
         /// </summary>
         [JsonIgnore]
+        [ObsoleteAttribute("Channels use their own voice regions. Guild region is deprecated. Please use the newer one on the voice channels.", false)]
```

Comment:
Write [Obsolete], not [ObsoleteAttribute].

Reply:
Done.